mars-sim-core/src/test/java/com/mars_sim/core/structure/building/BuildingCategoryTest.java

```diff
@@ -0,0 +1,47 @@
+package com.mars_sim.core.structure.building;
+
+
+import com.mars_sim.core.AbstractMarsSimUnitTest;
+import com.mars_sim.mapdata.location.BoundedObject;
+
+public class BuildingCategoryTest extends AbstractMarsSimUnitTest {
+    private static final BoundedObject BOUNDS = new BoundedObject(0, 0, 20, 10, 0D);
+
+    public void testGarage() {
+        assertBuildingType("Garage", BuildingCategory.VEHICLE);
+    }
+
+    public void testLargeGreenhouse() {
+        assertBuildingType("Large Greenhouse", BuildingCategory.FARMING);
+    }
+
+    public void testResearch() {
+        assertBuildingType("Research Hab", BuildingCategory.LABORATORY);
+    }
+    
+
+    public void testHallway() {
+        assertBuildingType("Hallway", BuildingCategory.CONNECTION);
+    }
+
+    public void testInfirmary() {
+        assertBuildingType("Infirmary", BuildingCategory.MEDICAL);
+    }
+   
+    public void testCommand() {
+        assertBuildingType("Command and Control", BuildingCategory.COMMAND);
+    }
+    
+    public void testQuarters() {
+        assertBuildingType("Residential Quarters", BuildingCategory.LIVING);
+    }
+    
```

Comment:
Let's also add testWorkshop(). 

We'll need to fix the order of WORKSHOP to go before LABORATORY in BuildingCategory.